### DISCUSSION TITLE
Embed Twitter posts

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#68918] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice to be able to incorporate a Twitter post into the issues.

Link example: https://x.com/github/status/1708136705843249510?s=20

This is the code provided by Twitter, to embed posts:
`<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Fun little app from <a href="https://twitter.com/garrrikkotua?ref_src=twsrc%5Etfw">@garrrikkotua</a> to create AI-generated Octocat logos with a simple prompt: <a href="https://t.co/oSdz6ozstF">https://t.co/oSdz6ozstF</a>.<br><br>We&#39;d love to see your creations. Share them below❣️ <a href="https://t.co/mS5hj7dmtk">pic.twitter.com/mS5hj7dmtk</a></p>&mdash; GitHub (@github) <a href="https://twitter.com/github/status/1708136705843249510?ref_src=twsrc%5Etfw">September 30, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>`

This is what you see when inserted:
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Fun little app from <a href="https://twitter.com/garrrikkotua?ref_src=twsrc%5Etfw">@garrrikkotua</a> to create AI-generated Octocat logos with a simple prompt: <a href="https://t.co/oSdz6ozstF">https://t.co/oSdz6ozstF</a>.<br><br>We&#39;d love to see your creations. Share them below❣️ <a href="https://t.co/mS5hj7dmtk">pic.twitter.com/mS5hj7dmtk</a></p>&mdash; GitHub (@github) <a href="https://twitter.com/github/status/1708136705843249510?ref_src=twsrc%5Etfw">September 30, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

**Comment by Angelk90** [comment]
Is there any news?

**Comment by 0PeterAdel** [comment]
# Embedding Twitter Posts in GitHub Issues

Including Twitter posts directly into GitHub issues could enhance context and interactivity. The standard Twitter embed code includes a blockquote and a script to load the tweet widget. However, GitHub issues currently have limitations with rendering external scripts, so the embed code may display only as plain text.

## Current Behavior

- **Display:** The tweet may appear as text with a link.
- **Interactivity:** Dynamic elements from the embed code (like interactive media) may not load.

## Workaround

- Use a direct link to the tweet so that users can click through to view it on Twitter.
- Consider adding a screenshot of the tweet alongside the link for better context.

## Future Considerations

The suggestion to support dynamic Twitter embeds in GitHub issues has been noted for potential future improvements.

Feedback is appreciated.
